docs/snippets/web-components/document-screen-fetch.js.mdx

```diff
@@ -0,0 +1,65 @@
+```js
```

Comment:
@kasperpeulen — Does this implementation look right to you?

Used here: https://storybook.js.org/docs/7.0/web-components/writing-stories/build-pages-with-storybook#mocking-rest-requests-with-msw-addon

Reply:
Looks good!